browser/src/UI/components/StatusBar.less

```diff
@@ -72,4 +73,48 @@
             }
         }
     }
+
+    .status-overflow(@hideWidth: 950px, @max: 15vw) {
+        .status-hide(@max){
+            overflow: hidden;
+            max-width: @max;
+            text-overflow: ellipsis;
+        }
+
+        @media(max-width: @hideWidth){
+            .status-hide(@max);
+            height: 100%;
+
+                    * > {
+                        .status-hide(100%)
+                    }
+                }
+            }
+
+            .status-disappear(@width) {
+                @media(max-width: @width){
+                    display: none !important;
+                }
+            }
+
+            .item-priority-0 {
+                .status-overflow(400px);
+            }
+
+            .item-priority-1 {
+                .status-overflow(650px, 35vw);
```

Comment:
Is there a reason we specify a max for priority 1, but not for the other cases? It's not clear to me what makes this one different.

Reply:
The practical effect of that line is that for example the filename component a '1' truncates earlier rather than dropping to half its size. The max here gives these items more space  before they collapse, the idea I was going for was that priority could be for larger components that we wanted on the screen for as long as possible